core/src/prediction/Prediction.cpp

```diff
@@ -41,14 +52,22 @@ const std::vector<double>& Prediction::get_error_estimates() const {
   return error_estimates;
 }
 
+const std::vector<double>& Prediction::get_excess_error_estimates() const {
+  return excess_error_estimates;
+}
+
 const bool Prediction::contains_variance_estimates() const {
   return !variance_estimates.empty();
 }
 
+const bool Prediction::contains_excess_error_estimates() const {
```

Comment:
Small comment, but I don't think we need this check as well, since we have the invariant that all error estimates contain both a debiased error, plus the bias.